stmt/stmts/insert.go

```diff
@@ -275,24 +198,19 @@ func (s *InsertIntoStmt) Exec(ctx context.Context) (_ rset.Recordset, err error)
 		}
 	}
 
-	txn, err := ctx.GetTxn(false)
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
-
 	for i, row := range rows {
 		// Notes: incompatible with mysql
 		// MySQL will set last insert id to the first row, as follows:
 		// `t(id int AUTO_INCREMENT, c1 int, PRIMARY KEY (id))`
 		// `insert t (c1) values(1),(2),(3);`
 		// Last insert id will be 1, not 3.
-		variable.GetSessionVars(ctx).SetLastInsertID(lastInsertIds[i])
```

Comment:
The behaviour of `LastInsertID` still differs from MYSQL, I will move this comments to a better place and fix the incompatible behaviour later.